pallets/migration/src/lib.rs

```diff
@@ -351,5 +388,34 @@ pub mod pallet {
 				.into(),
 			)
 		}
+
+		/// This extrinsic disables the call-filter. After this has been called the chain will accept
+		/// all calls again.
+		#[pallet::weight(<T as pallet::Config>::WeightInfo::finalize())]
+		#[transactional]
+		pub fn finalize(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin)?;
+
+			ensure!(
+				<Completed<T>>::get() == false,
+				Error::<T>::MigrationAlreadyCompleted
+			);
+
+			<Completed<T>>::put(true);
+
+			Self::deposit_event(Event::<T>::MigrationFinished);
+
+			Ok(())
+		}
+	}
+}
+
+impl<T: Config> Contains<<T as frame_system::Config>::Call> for Pallet<T> {
+	fn contains(c: &<T as frame_system::Config>::Call) -> bool {
+		if !pallet::Completed::<T>::get() {
+			T::Filter::contains(c)
+		} else {
+			true
+		}
```

Comment:
We can do some clean up here:

```suggestion		
		return pallet::Completed::<T>::get() || T::Filter::contains(c)
```

Reply:
I like. Will adapt with new filter enum.